src/cbc_sdk/platform/alerts.py

```diff
@@ -792,6 +793,23 @@ def _query_implementation(cls, cb, **kwargs):
         """
         return AlertSearchQuery(cls, cb).add_criteria("type", ["WATCHLIST"])
 
+    def get_watchlist_objects(self):
+        """
+        Returns the list of associated watchlist objects for the associated watchlist alert.
+
+        Example:
+            >>> watchlist_alert = cb.select(Alert, "f643d11f-59ab-478f-92c3-4198ca9b8230")
+            >>> watchlist_objects = watchlist_alert.get_watchlist_objects()
+
+        Returns:
+            Watchlist (list): A list of watchlist objects.
```

Comment:
Sorry to nitpick here, but this should be rather: `list: A list of watchlist objects`.

Reply:
Or rather `list[Watchlist]: A list of Watchlist objects.`